src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -39,13 +38,24 @@ const PatientInfo = ({ patientID, setName, setSchema }: IPatientInfoProps) => {
         }
     }
     return (
-        <div className="failed-container">
-            <Empty
-                description={
-                    <span>Fant ingen pasienter med dette personnummeret</span>
-                }
-            ></Empty>
-        </div>
+        <>
+            {!patientData && !patientDataError && (
+                <Row justify="space-around" align="middle">
+                    <Spin size="large" />
+                </Row>
+            )}
+            {patientDataError && (
+                <div className="failed-container">
+                    <Empty
+                        description={
+                            <span>
+                                Fant ingen pasienter med dette personnummeret
+                            </span>
+                        }
+                    ></Empty>
+                </div>
+            )}
+        </>
     );
```

Comment:
I think the message shown if this is true should be changed in order to provide more accurate feedback to the user. patientDataError implies that there was a transport error during the http request, not that the patient doesn't exist in the database.

Reply:
You mean to write something as 'Det finnes ingen pasienter med dette personnummeret'?